packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tabs/Tabs.js

```diff
@@ -58,7 +58,7 @@ function Tabs (props) {
     },
     tabs: {
       extend: `
-        button {
+        button[role="tab"] {
```

Comment:
This was breaking the radio button styling for the tutorial, so it needed greater specificity.